web/src/components/profile/PersonalizeYourTasksTab.test.tsx

```diff
@@ -0,0 +1,81 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+
+jest.mock("@/components/profile/ProfileTabHeader", () => ({
+  ProfileTabHeader: (() => <div data-testid="profile-tab-header" />) as React.FC,
+}));
+
+jest.mock("@/components/njwds-extended/Heading", () => ({
+  Heading: ({ children }: { children: React.ReactNode }): React.ReactElement => (
+    <div>{children}</div>
+  ),
+}));
+
+jest.mock("@/components/njwds/Icon", () => ({
+  Icon: (({ iconName }: { iconName: string }) => <span>{iconName}</span>) as React.FC<{
+    iconName: string;
+  }>,
+}));
+
+jest.mock("@/components/Content", () => ({
+  Content: (({ children }) => <div>{children}</div>) as React.FC<{ children: React.ReactNode }>,
+}));
+
+// Mock useConfig
+jest.mock("@/lib/data-hooks/useConfig");
+
+const mockConfig = {
+  profileDefaults: {
+    default: {
+      annualReportDeadlineHeader: "Annual Report Deadline",
+      annualReportDeadlineSubText: "You must file your report annually.",
+      filterCertificationsAndFundingHeader: "Certifications & Funding",
+      filterCertificationsAndFundingSubText: "Find out what you qualify for.",
+      locationBasedRequirementsHeader: "Location Requirements",
+      locationBasedRequirementsSubText: "Based on your business location.",
+    },
+    fields: {
+      nonEssentialQuestions: {
+        default: {
+          sectionHeader: "Non-Essential Questions",
+          sectionSubText: "Help us personalize your experience.",
+        },
+      },
+    },
+  },
+};
+
+describe("PersonalizeYourTasksTab", () => {
+  beforeEach(() => {
+    (useConfig as jest.Mock).mockReturnValue({ Config: mockConfig });
+  });
+
+  it("renders the tab panel with correct attributes", () => {
+    render(<PersonalizeYourTasksTab />);
+    const tabPanel = screen.getByRole("tabpanel");
+    expect(tabPanel).toHaveAttribute("id", "tabpanel-personalize");
+  });
+
+  it("renders the ProfileTabHeader", () => {
+    render(<PersonalizeYourTasksTab />);
+    expect(screen.getByTestId("profile-tab-header")).toBeInTheDocument();
+  });
+
+  it("renders all accordion headers with correct text", () => {
+    render(<PersonalizeYourTasksTab />);
+    expect(screen.getByText("Annual Report Deadline")).toBeInTheDocument();
+    expect(screen.getByText("Certifications & Funding")).toBeInTheDocument();
+    expect(screen.getByText("Location Requirements")).toBeInTheDocument();
+    expect(screen.getByText("Non-Essential Questions")).toBeInTheDocument();
+  });
+
+  it("renders accordion descriptions", () => {
+    render(<PersonalizeYourTasksTab />);
+    expect(screen.getByText("You must file your report annually.")).toBeInTheDocument();
+    expect(screen.getByText("Find out what you qualify for.")).toBeInTheDocument();
+    expect(screen.getByText("Based on your business location.")).toBeInTheDocument();
+    expect(screen.getByText("Help us personalize your experience.")).toBeInTheDocument();
```

Comment:
[sand] This should be referencing the content `Config` object.

Reply:
Done,